### ISSUE TITLE
Integrate /hal slack commands with Fuse Starter Java /hal endpoint

### ISSUE THREAD

**Issue by GalateaEbbs** [GalateaRaj/fuse-starter-java#371]
Issue Description
--
Context: https://github.com/galatea-associates/fuse-starter-java/issues/191

As part of reviving /hal commands in slack, we are rebuilding the functionality to exist in this project, where intents are determined via a wit.ai NLP application.

/hal commands currently still route to the google script and simply returns the response associated to the corresponding row. 

Now that we have some functionality stood up in fuse-starter-java with https://github.com/galatea-associates/fuse-starter-java/issues/189, we want to switch our /hal endpoints to call the live fuse-starter-java API at the /hal endpoint. 

Design
--

This involves 2 main tasks:
1. Stand up fuse-starter-java on an available server somewhere (decision needed, could be handled in another issue)
2. Change /hal commands to call the API living on that server.
3. 
Test Evidence
--
[include relevant tests and output. Should be complete before Review is approved]

Validation in Develop
-- 
[Proof issue is resolved in Develop. Should be complete before Issue is closed]
